meetings/2017-07-06.md

```diff
@@ -0,0 +1,98 @@
+# Node.js Foundation Community Committee Meeting 2017-07-08
+
+## Links
+
+* **GitHub Issue**: [community-committee#87](https://github.com/nodejs/community-committee/issues/87)
+* **Minutes Google Doc**: https://docs.google.com/document/d/1utEp06Pyk0uwJn77TWdxIpwuoiFRjHvKinSdbY1BwNk/
+* _Previous Minutes Google Doc_:
+https://docs.google.com/document/d/1GXbfPhfbLTr7g9Hp7YP-M60ux_Ua2z8XRXg__me4u4k/
+* _Video Recording_:
+https://www.youtube.com/watch?v=04JNTd_UXZw
+## Present
+
+* Bryan Hughes @nebrius (CommComm)
+* Gregor Martynus @gr2m (CommComm)
+* Jenn Turner @renrutnnej (observer)
+* Joe Sepi @joesepi (observer)
+* Olivia Hugger @pup (observer)
+* Richard Littauer @RichardLitt (observer)
+* Tierney Cyren @bnb (observer)
+* Tracy Hinds @hackygolucky (CommComm)
+* William Kapke @williamkapke (CommComm)
+
+## Agenda
+
+Extracted from **cc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/community-committee
+
+* Adding Jenn, Tierney, Michael, Olivia, and Richard as members [#77](https://github.com/nodejs/community-committee/pull/77)
+
+### nodejs/TSC
+
+* meta: moderation policy updates [#276](https://github.com/nodejs/TSC/pull/276)
+* Candidate "areas" for TSC and Community Committee  [#278](https://github.com/nodejs/TSC/issues/278)
+
+
+## Minutes
+
+### Adding Jenn, Tierney, Michael, Olivia, and Richard as members [#77](https://github.com/nodejs/community-committee/pull/77)
+
+**In-meeting votes:**
+* Tracy: +1
+* William: +1
+* Bryan: +1
+* Gregor: +1
+
+Voting will have to continue in the repo, in GitHub. Tracy [requested](https://github.com/nodejs/community-committee/pull/77#issuecomment-313516728) that members not present in the meeting vote.
+ 
+Discussion around 3 month limit on member inactivity, William clarified that this landed recently and is not applied retroactively.
```

Comment:
Let's clarify this a bit. We don't have a limit on activity, but rather a requirement that members need to participate. If a member does not participate within a three month time period, then they are converted from a member to an observer.